mezzanine/utils/cache.py

```diff
@@ -86,12 +85,11 @@ def flatten(seqs):
 def cache_key_prefix(request):
     """
     Cache key for Mezzanine's cache middleware. Adds the current
-    device and site ID.
+    site ID.
     """
-    cache_key = "%s.%s.%s." % (
+    cache_key = "%s.%s." % (
         settings.CACHE_MIDDLEWARE_KEY_PREFIX,
         current_site_id(),
-        device_from_request(request) or "default",
```

Comment:
Maybe we just leave the string "default" here with a note in the comment - I think for most people that would mean their cache keys stay the same.